composer.json

```diff
@@ -1,12 +1,12 @@
 {
-    "name": "evodelavega/activecampaign-api-php",
+    "name": "ravimadan/activecampaign-api-php",
     "description": "An alternative, more PSR-compliant AC API wrapper",
     "license": "MIT",
     "keywords": ["activecampaign", "email-marketing", "newsletter", "marketing-automation", "subscribe", "forms", "emails", "automation"],
     "authors": [
         {
-            "name": "Elias Van Ootegem",
-            "role": "Developer"
+            "name": "Ravi Madan",
```

Comment:
Rather than replace, maybe add a second author. I'm fine with crediting anyone and everyone who worked on this, myself included. I'm not working with PHP anymore though, so if you'd prefer, I'm more than happy giving you ownership over this repo